sky/sdk/example/widgets/navigation.dart

```diff
@@ -21,8 +21,10 @@ List<Route> routes = [
         new RaisedButton(
           child: new Text('START ADVENTURE'),
           onPressed: () => navigator.pushNamed('adventure')
-        )
-      ])
+        )],
```

Comment:
This ends up not aligned. I think you really want the children indented 4 from the 'child', then the other arguments (and the closing "]") indented 2.